application/src/main/java/org/exoplatform/chat/portlet/notification/templates/index.gtmpl

```diff
@@ -10,6 +10,7 @@
   data-chat-interval-notif="<%=chatIntervalNotif%>"
   data-chat-interval-status="<%=chatIntervalStatus%>"
   data-short-space-name="<%=shortSpaceName%>"
+  data-db-name="<%=dbName%>"
```

Comment:
The database name is set in the DOM of the page, so anybody could change it (with Firebug for example). This is a big security issue. We need to find a safer way to pass the database name.